org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/jsts/ui/messages.properties

```diff
@@ -0,0 +1,24 @@
+#/*******************************************************************************
+# * Copyright (c) 2022 Red Hat Inc. and others.
+# * This program and the accompanying materials are made
+# * available under the terms of the Eclipse Public License 2.0
+# * which is available at https://www.eclipse.org/legal/epl-2.0/
+# *
+# * SPDX-License-Identifier: EPL-2.0
+# *
+# * Contributors:
+# *  Angelo ZERR (Red Hat Inc.) - initial implementation
+# *******************************************************************************/
+
+# TypeScript Inlay Hints preference page
+TypeScriptInlayHintPreferencePage_includeInlayEnumMemberValueHints=Enable/disable inlay hints for member values in enum declarations.
```

Comment:
Please replace Enable/Disable with "Show".
Also please try to factorize the label "Show inlay hints for..."
so the checkbox items become only "member vaues in enum declaration", "implicity return ty[e on function signatures", and so on.

Reply:
I tried to improve quickly the UI, do you like?

![image](https://user-images.githubusercontent.com/1932211/201672587-db67dab3-9c89-4ef5-9aa0-b4b17c837192.png)